elliptic-curve/src/lib.rs

```diff
@@ -5,7 +5,7 @@
     html_logo_url = "https://raw.githubusercontent.com/RustCrypto/media/8f1a9894/logo.svg",
     html_favicon_url = "https://raw.githubusercontent.com/RustCrypto/media/8f1a9894/logo.svg"
 )]
-#![forbid(unsafe_code)]
+#![deny(unsafe_code)]
```

Comment:
It'd be good to leave a comment here about what unsafety is allowed

```suggestion
#![deny(unsafe_code)] // only for newtype casting
```

...or something to that effect

Reply:
Done.